auction-deploy/auction_deploy/cli.py

```diff
@@ -330,7 +330,8 @@ def status(auction_address, jsonrpc):
     locker_address = contracts.locker.address
     slasher_address = contracts.slasher.address
     locker_initialized = contracts.locker.functions.initialized().call()
-    slasher_initialized = contracts.slasher.functions.initialized().call()
+    if locker_initialized:
```

Comment:
This is not the right way to fix this. Why do you check for locker stuff to call then something on slasher. This is you using to much knowledge of how it is implemented. Please fix it by checking stuff on the slasher contract.

Reply:
I initially had a check that the slasher address is `0`, but this relies on the fact that the address of a contract we did not find is `0`. Is that better?

Reply:
what do you mean we did not find? I though it is 0x if we did not initialized it?

Reply:
I think `get_deployed_auction_contracts` should do those check already, and only set a contract if it has a proper address, otherwise `None` that you can check here if contracts.slasher is None.